src/main/java/com/intuit/graphql/orchestrator/batch/VariableDefinitionFilter.java

```diff
@@ -54,6 +63,15 @@ public Set<String> getVariableReferencesFromNode(GraphQLSchema graphQLSchema, Gr
         .root(rootNode)
         .build();
 
+    QueryTransformer queryTransformer = QueryTransformer.newQueryTransformer()
```

Comment:
Not sure if I missed it, but where is `queryTransformer` used?

Reply:
Good catch that was used during testing; removing that instance now